perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -1544,23 +1544,27 @@ public List<Group> call(ApiCaller ac, Deserializer parms) throws PerunException{
 	 * @param vo int <code>id</code> of vo
 	 * @param fromIndex int begin index of returned subList, included
 	 * @param attrNames List<String> if attrNames is null method will return RichGroups containing all attributes
+	 * @param roles list of selected roles (if empty, then return groups by all roles)
+	 * @param types list of selected types of roles (if empty, then return by roles of all types)
```

Comment:
I am not sure but I think that you should duplicate the javadoc for the new parameters, not just change the old one - from this javadoc is created the documentation and to keep in the docs also old methods with fewer parameters you should probably duplicate it. (it should be applied for all methods you changed in this file)